code/modules/mob/living/carbon/xenomorph/castes/hivemind/hivemind.dm

```diff
@@ -19,22 +19,25 @@
 	tier = XENO_TIER_ZERO
 	upgrade = XENO_UPGRADE_ZERO
 
-	mouse_opacity = MOUSE_OPACITY_OPAQUE
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
 	see_invisible = SEE_INVISIBLE_LIVING
-	invisibility = INVISIBILITY_OBSERVER
+	invisibility = INVISIBILITY_ABSTRACT
```

Comment:
This just makes it harder for observers to see the hivemind flying around. The hud is still left without the inner mob icon.

Reply:
fixed pbing/clicking on it as a mob